Download binary on Release

I failed trying to compile darling into my server at digital ocean. It ran out of disk space. 
Would it be possible to provide pre-built binaries, either verified or provided by a CI-CD service?

@ahyattdev While I may be the last one to touch a file in there, I've only edited one line in there. I know far less about deb packaging than rpms.

```
docker run -it --rm -v `pwd`:/src debian:9
# Following https://wiki.darlinghq.org/build_instructions, debian 8 didn't work due to some weird freetype conflict, but that was after I did dpkg --add-architecture i386, so maybe I just screwed up that test altogether
apt install cmake clang-4.0 bison flex xz-utils libfuse-dev libudev-dev pkg-config libc6-dev-i386 linux-headers-amd64 libcap2-bin git libcairo2-dev libgl1-mesa-dev libtiff5-dev libfreetype6-dev libxml2-dev libegl1-mesa-dev libfontconfig1-dev libbsd-dev build-essential devscripts clang ninja-build
cd /src
./tools/makedeb
```

And... I get the exact same problem!
I've never used `dch` so I don't have a clue how it is supposed to be used.

Theory: Fix the changelog to say `darling` instead of `PACKAGE`.

- Adding `--package darling` to the `dch` command works, but I have no idea how compatible the flag is in debian-land

```
cd /src
./tools/makedeb
```

Got further before crashing

```
make[1]: Entering directory '/src'
mkdir debbuild
cd debbuild && CFLAGS="" CXXFLAGS="" CPPFLAGS="" LDFLAGS="" cmake .. -DCMAKE_TOOLCHAIN_FILE=../Toolchain.cmake -DCMAKE_INSTALL_PREFIX=/usr
CMake Error at /usr/share/cmake-3.7/Modules/CMakeDetermineSystem.cmake:94 (message):
  Could not find toolchain file: ../Toolchain.cmake
```

At this point, I don't know why `rules` references a  `Toolchain.cmake` file that does not exist? I don't use that flag in the RPMs. Sorry I can't be of more help.

--- a/.gitignore
+++ b/.gitignore
@@ -38,3 +38,5 @@ build
 rpm/SOURCES
 rpm/RPMS
 rpm/SRPMS
+
+debbuild

--- a/debian/rules
+++ b/debian/rules
@@ -4,7 +4,7 @@
 
 override_dh_auto_configure:
 	mkdir debbuild
-	cd debbuild && CFLAGS="" CXXFLAGS="" CPPFLAGS="" LDFLAGS="" cmake .. -DCMAKE_TOOLCHAIN_FILE=../Toolchain.cmake -DCMAKE_INSTALL_PREFIX=/usr
+	cd debbuild && CFLAGS="" CXXFLAGS="" CPPFLAGS="" LDFLAGS="" cmake .. -DCMAKE_INSTALL_PREFIX=/usr
 
 override_dh_auto_build:
 	cd debbuild && $(MAKE) && make lkm_generate printwrapgen

--- a/tools/makedeb
+++ b/tools/makedeb
@@ -3,6 +3,6 @@ set -e
 
 DATE=$(date +%Y%m%d)
 rm -f debian/changelog
-EDITOR=true NAME="Darling build server" EMAIL=noreply@darlinghq.org dch -u low -v 0.1.${DATE}-1 --create --distribution testing "Daily build"
+EDITOR=true NAME="Darling build server" EMAIL=noreply@darlinghq.org dch -u low -v 0.1.${DATE}-1 --create --distribution testing --package darling "Daily build"
 debuild -us -uc -b
 

